src/ape/api/address.py

```diff
@@ -24,6 +24,9 @@ def provider(self) -> ProviderAPI:
     def provider(self, value: ProviderAPI):
         self._provider = value
 
+    def _create_transaction(self, **kwargs):
+        return self.provider.network.ecosystem.create_transaction(**kwargs)
+
```

Comment:
This actually seems to make the most sense under `AccountAPI`, since a generic address (or contract account) cannot produce a transaction